README.md

```diff
@@ -5,10 +5,13 @@
 
 > Docker Compose file for running the entire OpenTOSCA stack.
 
+:warning: On newer docker installations `docker-compose` will be integrated into the docker cmd. If this is the case then all `docker-compose` commands must be written as `docker compose` (without the hyphen)!
```

Comment:
Maybe with a link to the annoucement?

Reply:
Docker Compose is already integrated as beta into Docker For Mac as example.
However, I don't think you _must_ use it right?

Reply:
Bei mir gab es kein `docker-compose` command mehr. Da ich docker desktop 4.x habe ist das vielleicht jetzt neu, hab aber auch nicht weiter gesucht.